pkg/webhook/resources/persistentvolumeclaim/validator.go

```diff
@@ -150,3 +155,24 @@ func (v *pvcValidator) Update(_ *types.Request, oldObj runtime.Object, newObj ru
 
 	return nil
 }
+
+func validateAnnotation(pvc *corev1.PersistentVolumeClaim) error {
+	if pvc == nil || pvc.Annotations == nil {
+		return nil
+	}
+
+	snapshotMaxCount, snapshotMaxSize, err := util.GetSnapshotMaxCountAndSize(pvc)
+	if err != nil {
+		return werror.NewInvalidError(err.Error(), "annotations")
+	}
+
+	if snapshotMaxCount < 2 || snapshotMaxCount > 250 {
+		return werror.NewInvalidError("snapshot max count should be between 2 and 250", fmt.Sprintf("annotations[%s]", util.AnnotationSnapshotMaxCount))
+	}
```

Comment:
Should this actually be the value of `settings.DefaultSnapshotMaxCount.GetInt()`, rather than hard-coding 250?